tools/harness_capability_profiles.py

```
@@ -0,0 +1,869 @@
+    if non_empty_string(profile.get("evidence_category")) and profile.get("evidence_category") not in EVIDENCE_CATEGORIES:
+        results.append(CheckResult(f"profile:{harness_id}:evidence_category", False, "invalid evidence category", relative_path.as_posix()))
+    if not string_list(profile.get("components")):
+        results.append(CheckResult(f"profile:{harness_id}:components", False, "components must be a list of strings", relative_path.as_posix()))
+    if not string_list(profile.get("local_observations")):
+        results.append(
+            CheckResult(f"profile:{harness_id}:local_observations", False, "local_observations must be a list of strings", relative_path.as_posix())
+        )
+
+    scope = profile.get("scope")
+    if not isinstance(scope, dict) or scope.get("surface") != "vanilla_harness_capability_surface":
+        results.append(CheckResult(f"profile:{harness_id}:scope", False, "scope must describe vanilla harness surface", relative_path.as_posix()))
```

Ruff (0.15.12)

[warning] Boolean positional value in function call

(FBT003)